src/Application/Application.csproj

```diff
@@ -1,5 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <PropertyGroup>
+    <RootNamespace>CleanCompanyName.DDDMicroservice.Application</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="Common\Validators\**" />
+    <EmbeddedResource Remove="Common\Validators\**" />
+    <None Remove="Common\Validators\**" />
+  </ItemGroup>
```

Comment:
It looks like you have an empty folder in the solution? If you delete it, you could remove those lines from `.csproj` file.

Reply:
Yes, I didn't notice that when do not include items on a project, this change is marked as removed on the project.